src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
```

Comment:
None of our [modular sizes](https://github.com/cloudfour/cloudfour.com-patterns/blob/master/src/assets/toolkit/styles/base/_config.css#L98-L101) worked for this?